src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,23 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      <UserInfo user={post.user} />
+    </div>
+
+    <p className="PostInfo__body">
+      {post.body}
+    </p>
+    {post.comments.length >= 1 ? (
+      <CommentList comments={post.comments} />
+    ) : (
+      <b data-cy="NoCommentsMessage">
+        {/* */}
+      </b>
+    )}
```

Comment:
1. Let's add empty line between sibling elements
```
<p className="PostInfo__body">
  {post.body}
</p>

{post.comments.length >= 1 ? (
  <CommentList comments={post.comments} />
) : (
  <b data-cy="NoCommentsMessage">
    {/* */}
  </b>
)}
```

You forgot to add `No comments yet` message and `<hr />` tag before it